docs/docs/dev-manuals/services/catalog.mdx

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 1
+---
+
+import CustomImage from "@site/src/components/CustomImage";
+
+# Catalog Service
+
+The catalog service consists of the bounded contexts `Catalog`, `Category`, and `CategoryValue`.
+It is responsible for managing products, their variants and versions, categories, and their respective characteristics and values.
+
+## Domain Model
+
+<CustomImage path="/renderedDiagrams/catalogDomainModel" width="1240.06640625" height="946.625" />
+
+:::info
+
+Imported entities from other bounded contexts are marked with a dashed border.
+
+:::
+
+## API
+
+The service provides a GraphQL API, which is documented in the [GraphQL API documentation](/docs/graphql/catalog).
+
+## Technology Stack
+
+- **Language**: Kotlin
+- **Framework**: Spring Boot
+- **GraphQL Library**: GraphQL Kotlin
+- **Database**: PostgreSQL
+- **Schema Management**: Flyway
+
+## Repository Structure
+
+The repository is structured as follows:
+
+<div className="repository-structure" >
+
+- `/src/main/`: Source code of the service
+  - `kotlin/org/misarch/catalog/`: Kotlin source code
+    - `event/`: Dapr event handling
+      - `model/`: Event DTOs
+    - `graphql/`: GraphQL API
+      - `dataloader/`: Data loaders for efficient data fetching
+      - `federation/`: Apollo Federation entitiy resolvers
+      - `input/`: Input types for mutations
+      - `model/`: GraphQL types
+        - `connection/`: Connection types and logic for paginated queries
+    - `persistance/`: Database entities and repositories
+      - `model/`: R2DBC entities
+      - `repository/`: Spring Data R2DBC repositories
+    - `service/`: Service logic
+  - `resources/`: Spring Boot application configuration and SQL schema migration files
+    - `db/migration/`: Flyway SQL schema migration files
+
+</div>
+
+## Defined Events
```

Comment:
```suggestion
## Events
```

Reply:
I don't really get this change.
Defined event means that this service has the authority over the schema of the event.
This is also communicated by the topic starting with the name of the service.

Reply:
Yes, I'm simply arguing that this `defined` is redundant, after all which other events could you mean?
But yes, if you want to keep this technicality no one except you knows about, then okay.

Reply:
the other events are the events defined by other services this service uses in some way, thus I leave it like that